docs/_pages/playbooks.md

```diff
@@ -36,3 +38,6 @@ sidebar:
 | [Risk Notable Protect Assets and Users](/playbooks/risk_notable_protect_assets_and_users/)| None | Response |
 | [Risk Notable Review Indicators](/playbooks/risk_notable_review_indicators/)| None | Response |
 | [Risk Notable Verdict](/playbooks/risk_notable_verdict/)| None | Response |
+| [Start Investigation](/playbooks/start_investigation/)| None | Investigation |
+| [Threat Intel Investigate](/playbooks/threat_intel_investigate/)| None | Investigation |
+| [TruSTAR Enrich Indicators](/playbooks/trustar_enrich_indicators/)| None | Input |
```

Comment:
Type should be Investigation I think.

Reply:
FIxed the Input playbooks to fit in Investigation vs Response.